src/js/ui/shared/form.ts

```diff
@@ -13,6 +14,21 @@ function renderOption(label: string, value: string, storedValue: string, labelAr
   }, i18n(label, labelArg, labelArg2));
 }
 
+
+function renderOptionGroup(label:string, value:string | SelectOptionGroup, storedValue:string, labelArg:string, labelArg2:string): Mithril.Children {
+  if (typeof value === 'string') {
```

Comment:
that's right, type guards are simple predicate functions with a specific return type annotation